### ISSUE TITLE
New live context not working

### ISSUE THREAD

**Issue by Oracle-of-Delfos** [pieces-app/support#212]
### Software

Desktop Application

### Operating System

Windows

### Your Pieces OS Version

v9.0.4

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

Just tried to use the new live context,but as soon as i enable the WPE engine, pieces desktop immediately crashes saying that pieces os isnt running, and when it boot up again the WPE engine is still off.
Here's a conversation i had with one of the mods from discord with more details and screenshots: https://discord.com/channels/1098986658851999776/1241169776923906209

**Comment by HRA42** [comment]
I have the same problem. Here are the logs files:
[log-05182024.txt](https://github.com/pieces-app/support/files/15366335/log-05182024.txt)
[log-05182024.txt](https://github.com/pieces-app/support/files/15366337/log-05182024.txt)

**Comment by mark-at-pieces** [comment]
Taking a look into this right now: my initial guess is that this is related to Live Context. Will be digging and will report back shorty. Thank you @HRA42 && @Oracle-of-Delfos for bringing this to our attention

**Comment by mark-at-pieces** [comment]
@HRA42 have you been able to successfully use in the past? and if so is this the same computer that you have used this for?

**Comment by HRA42** [comment]
@mark-at-pieces Yes on my Mac it works perfectly. On Windows I have the same issue with my main pc and all VMs I try to run the live context inside.

**Comment by mark-at-pieces** [comment]
We have narrowed in on the cause of the Problem and appears to be language(English) server related.

So @HRA42's computer was set to German and the WPE would crash everytime, untill his was switched to english and everything is working now.

Will continue to update this channel here with status updates expect a production patch to resolve this issue!

Thank you to @HRA42 for taking the time to help us solve this issue!

**Comment by HRA42** [comment]
Are there any updates on this? Do you guys need help or want to test anything further? 

If I can do anything to help you, feel free to hit me up on this!

**Comment by mark-at-pieces** [comment]
> Are there any updates on this? Do you guys need help or want to test anything further?
> 
> If I can do anything to help you, feel free to hit me up on this!

Hey @HRA42, 

Thanks for checking in, we have the solution ready to rock.

This will be going out in our next release!

Planning to get a release out Thursday by the latest.

**Comment by mark-at-pieces** [comment]
@HRA42 This is now live w/ 9.0.5! lemme know if there are any regressions

**Comment by HRA42** [comment]
thanks, for me its working :) @Oracle-of-Delfos maybe you can confirm too?

**Comment by Oracle-of-Delfos** [comment]
Oh yeah,just tested it today, it's working wonderfully, gonna try implementing it together with my obsidian :)
